aat/src/aat/resources/features/F-094 NoC ChallengeQuestions/F-094.feature

```diff
@@ -0,0 +1,38 @@
+@F-094
+Feature: F-094 Retrieve ChallengeQuestions for Notice of Change
+
+  Background:
+    Given an appropriate test context as detailed in the test data source,
+    And a user [MCA System - with an IDAM role of caseworker CAA],
+
+  @S-094.1
+  Scenario: Must successfully return the contents of the ChallengeQuestion definition
```

Comment:
Suggestion: `must successfully return challenge questions of a correctly configured case type`